server/webapp/WEB-INF/rails.new/lib/encryption_api_rate_limiter.rb

```diff
@@ -0,0 +1,31 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+require 'rack/throttle'
+
+class EncryptionApiRateLimiter < Rack::Throttle::Minute
+
+  def allowed?(request)
+    if request.url.include?("/go/api/admin/encrypt/")
+      super
+    else
+      true
+    end
+  end
+
+  def client_identifier(request)
+    com.thoughtworks.go.server.util.UserHelper.getUserName().getUsername().to_s
+  end
+end
```

Comment:
The expected content type is JSON, while this limiter prints `text/plain` with the message `Rate Limit Exceeded`. Consider using overriding `http_error(code, msg, headers)`